bin/digital-ocean/setup

```diff
@@ -0,0 +1,266 @@
+bundle install
+## Compile assets:
+bundle exec rake assets:precompile
+
+chown -R rails: /home/rails/rails_project
+
+## Setup Nginx
+# --------------------------------------------------------------
+apt-get -y install nginx
+cat <<EOT >> /etc/nginx/sites-available/rails
+upstream app_server {
+    server unix:/var/run/unicorn.sock fail_timeout=0;
+}
+
+server {
+    listen   80;
+    root /home/rails/rails_project/public;
+    server_name _;
+    index index.htm index.html;
+
+    location / {
+            try_files \$uri/index.html \$uri.html \$uri @app;
+    }
+
+    location ~* ^.+\.(jpg|jpeg|gif|png|ico|zip|tgz|gz|rar|bz2|doc|xls|exe|pdf|ppt|txt|tar|mid|midi|wav|bmp|rtf|mp3|flv|mpeg|avi)\$ {
+                    try_files \$uri @app;
+            }
+
+     location @app {
+            proxy_set_header X-Forwarded-For \$proxy_add_x_forwarded_for;
+            proxy_set_header Host \$http_host;
+            proxy_redirect off;
+            proxy_pass http://app_server;
+    }
+}
+EOT
+rm /etc/nginx/sites-enabled/default
+ln -s /etc/nginx/sites-available/rails /etc/nginx/sites-enabled/rails
+# --------------------------------------------------------------
+
+## Unicorn
+# --------------------------------------------------------------
+mkdir /var/log/unicorn/
+cat <<EOT >> /etc/unicorn.conf
+listen "unix:/var/run/unicorn.sock"
+worker_processes 4
+user "rails"
+working_directory "/home/rails/rails_project"
+pid "/var/run/unicorn.pid"
+stderr_path "/var/log/unicorn/unicorn.log"
+stdout_path "/var/log/unicorn/unicorn.log"
+EOT
+
+SECRET_KEY_BASE=$(rake -f /home/rails/rails_project/Rakefile secret)
+cat <<EOT >> /etc/default/rails_project
+# Generate by running \`rake -f /home/rails/rails_project/Rakefile secret\`
+export SECRET_KEY_BASE=$SECRET_KEY_BASE
+export MONGODB_URI=mongodb://rails:$MONGO_PASS@127.0.0.1:27017/venice
+export HOST=$HOST
+export SMTP_ADDRESS=placeholder
+export SMTP_DOMAIN=placeholder
+export SMTP_PASSWORD=placeholder
+export SMTP_USERNAME=placeholder
+EOT
+chmod +x /etc/default/rails_project
+
+cat <<EOT >> /etc/default/unicorn
+# Change paramentres below to appropriate values and set CONFIGURED to yes.
+CONFIGURED=yes
+
+# Default timeout until child process is killed during server upgrade,
+# it has *no* relation to option "timeout" in server's config.rb.
+TIMEOUT=30
+
+# Path to your web application, sh'ld be also set in server's config.rb,
+# option "working_directory". Rack's config.ru is located here.
+APP_ROOT=/home/rails/rails_project
+
+# Server's config.rb, it's not a rack's config.ru
+CONFIG_RB=/etc/unicorn.conf
+
+# Where to store PID, sh'ld be also set in server's config.rb, option "pid".
+PID=/var/run/unicorn.pid
+RAILS_ENV="production"
+UNICORN_OPTS="-D -c \$CONFIG_RB -E \$RAILS_ENV"
+
+RUBY_VERSION=$RUBY_VERSION
+PATH=/usr/local/rvm/rubies/\$RUBY_VERSION/bin:/usr/local/sbin:/usr/bin:/bin:/sbin:/usr/local/rvm/bin:/usr/local/rvm/gems/\$RUBY_VERSION@global/bin:/usr/local/rvm/gems/\$RUBY_VERSION/bin/
```

Comment:
Let's try to replace PATH, GEM_HOME and GEM_PATH definition with with `rvm use $RUBY_VERSION` — I guess it does the same.